test/suites/ubuntu-core-edgex-image/main.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+)
+
+func main() {
+	scripts := []string{"setup.sh", "build.sh", "run.sh"}
+
+	for _, s := range scripts {
+		cmd := exec.Command("bash", s)
+		cmd.Stdout = os.Stdout
+		cmd.Stderr = os.Stderr
+		err := cmd.Run()
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Failed to execute %s: %s\n", s, err)
+			os.Exit(1)
+		}
+	}
+
+	// todo: find a way to execute test.sh after compeleting Ubuntu Core Setup in run.sh script
+	// cmd := exec.Command("bash", "test.sh")
+	// cmd.Stdout = os.Stdout
+	// cmd.Stderr = os.Stderr
+	// if err := cmd.Run(); err != nil {
+    // 	fmt.Fprintf(os.Stderr, "Failed to execute test.sh: %s\n", err)
+   	// 	os.Exit(1)
+	// }
```

Comment:
This is problematic for two reasons:
1. The setup makes changes to the host system and leaves them there (no cleanup).
2. It is not a go test, like others testing suites

I suggest not dealing with this right now and only make a Go test wrapper for the bash scripts if we decide to add these tests in a CI workflow.

Reply:
Make sense, removed.